src/pages/bulkImport/utils/useBulkImportFields.js

```diff
@@ -87,10 +98,24 @@ export default function useBulkImportFields() {
       );
       return bulkEncounterFields.map(f => {
         const additionalProperties = {};
+        if (
+          f?.fieldType === 'select' ||
+          f?.fieldType === 'multiselect'
+        ) {
+          additionalProperties.type = 'select';
+        }
+        if (f?.choices) additionalProperties.options = f.choices;
         if (f.name === 'taxonomy') {
           additionalProperties.type = 'select';
           additionalProperties.options = speciesOptions;
         }
+        if (f.name === 'sex') {
+          const sanitizedSexOptions = Object.values(
+            omitBy(sexOptions, sex => sex?.labelId === 'BLANK'),
```

Comment:
Had to remove Blank from the list, because a value of null was giving a type error.